app/src/main/java/com/nytimes/android/sample/SampleApp.java

```diff
@@ -59,11 +60,11 @@ private void initPersister() {
         return StoreBuilder.<RedditData>barcode()
                 .fetcher(barCode -> provideRetrofit().fetchSubreddit(barCode.getKey(), "10"))
                 .memoryPolicy(
-                    MemoryPolicy
-                        .builder()
-                        .setExpireAfter(10)
-                        .setExpireAfterTimeUnit(TimeUnit.SECONDS)
-                        .build()
+                        MemoryPolicy
```

Comment:
Sanity check. Is this spacing change intentional? Looks like it was indented already